projects/demo/src/modules/animations/examples/fade-in/index.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, Input} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {tuiFadeIn} from '@taiga-ui/core';
+import {from, of} from 'rxjs';
+import {concatMap, delay, repeat, startWith} from 'rxjs/operators';
+
+@Component({
+    selector: 'tui-fade-in-example',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+    animations: [tuiFadeIn],
+})
+export class TuiFadeInExample {
+    @Input()
+    speed = 0;
+
+    isShown$ = from([false, true]).pipe(
+        concatMap(val => of(val).pipe(delay(1.5 * this.speed))),
+        repeat(),
+        startWith(true),
+    );
+
+    get animation() {
+        return {
+            value: '',
```

Comment:
nit: it's just an example but don't you want to add speed as argument and add tuiPure to avoid object re-creation?

Reply:
fixed